libs/sdk-ui-kit/src/Header/typings.ts

```diff
@@ -46,6 +46,7 @@ export interface IAppHeaderProps {
 
     userName: string;
 
+    expiredDate?: Date;
```

Comment:
Just a confirmation: the type of the expiration date in the entitlement is string, isn't it? If we mark the type of prop `expiredDate` as Date, we have to write code in each app to convert string to Date before passing it to Header component, don't we?

→ Should we define the type of `expiredDate` as string instead, and then in function `getTrialCountdown` we write some additional codes to convert string to date. With this approach, we don't have to duplicate the code to convert string to date in multiple apps, do we?

Reply:
Fixed. Please check again. Thank you!